ReactAndroid/src/main/java/com/facebook/react/uimanager/events/EventDispatcher.java

```diff
@@ -129,6 +130,24 @@ public void dispatchEvent(Event event) {
       // touch event dispatch will hit this codepath, and we simply queue them so that they
       // are dispatched once ReactContext creation completes and JS app is running.
     }
+
+    for (EventDispatcherListener listener : mListeners) {
+      listener.onEventDispatched(event);
```

Comment:
I mentioned that one earlier and not sure if you responded for my comment, but cannot find your response.

I think it would be better if we don't pass events that are handled natively to JS. That event is not going to be utilised on JS side and will just bloat the bridge.